<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Отрезок 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Отрезок 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>13</y>
     <width>47</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>x1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line1_x1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>13</y>
     <width>47</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>x1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line2_x1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>53</y>
     <width>47</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>y1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line1_y1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>53</y>
     <width>47</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>y1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line2_y1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>93</y>
     <width>47</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>x2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line1_x2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>133</y>
     <width>47</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>y2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line1_y2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>93</y>
     <width>47</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>x2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line2_x2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>133</y>
     <width>47</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>y2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line2_y2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="drawLines">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Нарисовать линии</string>
   </property>
  </widget>
  <widget class="QPushButton" name="findCommonPoint">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Найти общую точку</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Координаты общей точки</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="commonPoint_y">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>243</y>
     <width>47</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>y</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>243</y>
     <width>47</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="commonPoint_x">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="point_y">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>27</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>10</y>
     <width>47</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>y</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="point_x">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>27</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>10</y>
     <width>47</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QPushButton" name="drawPoint">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>60</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Нарисовать точку</string>
   </property>
  </widget>
  <widget class="QPushButton" name="findDistance">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>82</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Найти дистанцию</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
